File: DemoSelenium/DemoSeleniumPython/TesteSimples.py
import unittest
from selenium import webdriver

class TesteSimples(unittest.TestCase):

    global driver;

    def setUp(self):
        self.driver = webdriver.Firefox();

    def test_acesso(self):
        self.driver.get("http://seleniumhq.org")

    def tearDown(self):
        self.driver.quit

if __name__ == '__main__':
    unittest.main()

File: DemoSelenium/DemoSeleniumRuby/TesteSimples.rb
require "rubygems"
require "selenium-webdriver"
require "minitest/autorun"

class TesteSimples < MiniTest::Test

    # create selenium objects
    def setup
        @driver = Selenium::WebDriver.for :firefox
    end

    def test_post
        @driver.get "http://globo.com"
    end

    def teardown
        @wait = Selenium::WebDriver::Wait.new :timeout => 5
        @driver.quit  
    end

end
